### PR TITLE
hint about missing functionality in AdvOption

### PR DESCRIPTION
**Pull request by Laserlicht**
- add info for not implemented replay
- reorder button code
- fix exit position (1 pixel to left)

### CODE REVIEW

File: Mods/vcmi/config/vcmi/english.json [hunk 1]
```diff
@@ -72,7 +73,6 @@
 	"vcmi.lobby.noUnderground" : "no underground",
 	"vcmi.lobby.sortDate" : "Sorts maps by change date",
 
-	"vcmi.client.errors.invalidMap" : "{Invalid map or campaign}\n\nFailed to start game! Selected map or campaign might be invalid or corrupted. Reason:\n%s",
```
Comment (dydzio0614) [inline Mods/vcmi/config/vcmi/english.json]: why was this removed?

Comment (Laserlicht) [reply Mods/vcmi/config/vcmi/english.json]: Was an mistake. See the full change history...
